public/locales/en/translations.json

```diff
@@ -9,5 +9,11 @@
     "devices": "devices",
     "rules": "rules",
     "maintenance": "maintenance"
+  },
+  "components": {
```

Comment:
I would remove the components key, just use devices.

Reply:
OK.